libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationActivity.java

```diff
@@ -76,6 +77,7 @@ public void onNavigationReady() {
       extractCoordinates(options);
     }
     extractConfiguration(options);
+    options.navigationOptions(MapboxNavigationOptions.builder().locale(LocaleUtils.getLocale(this)).build());
```

Comment:
Can we extract this into a private method?  Something like `extractLocale` to align with the other methods here?